_data/projects.yml

```diff
@@ -58,34 +58,35 @@ projects:
-Unfortunately, the act of performing an update can itself be a
-way to attack a computer, if a malicious update is installed. TUF is a
-comprehensive, flexible framework to secure software updates even in situations
-where the software repository is compromised. Developers can integrate TUF
-into any software update system, or native programming language due to its
-concise, self-contained architecture and specification. It offers both developers
-and users protection from a host of potential attacks."
+      description: "Software repositories can receive updates on software or
+project metadata as frequently as every few minutes. This growing flow of updates
+has created a large and potentially devastating new attack surface for malicious
+actors, as malware submitted through this process offers the opportunity to
+impact thousands of users at a time. The Update Framework (TUF) is a comprehensive,
+flexible approach to securing software updates, even in situations where the repository
+is compromised. Developers can integrate TUF into any software update system, or
+native programming language due to its concise, self-contained architecture and
+specification. It’s adoption in 2017 by the <a href=\"https://www.cncf.io\">Cloud Native Computing Foundation</a>
+of the <a href=\"https://www.linuxfoundation.org/\">Linux Foundation</a> along
+with Docker's popular implementation of the framework, now makes TUF the de facto
+security standard for the cloud."
+
       products: "<a href=\"https://theupdateframework.com\">TUF</a> is used in
-      production by a <a 
-href=\"https://theupdateframework.github.io/adoptions.html\">variety of 
+      production by a <a href=\"https://theupdateframework.github.io/adoptions.html\">variety of
 projects</a>, including <a href=\"https://www.vmware.com/\">VMware</a>,
-      <a href=\"https://blog.cloudflare.com/pal-a-container-identity-bootstrapping-tool/\">Cloudflare</a>, and
-      <a href=\"https://blog.docker.com/2015/08/content-trust-docker-1-8/\">Docker</a>.
-     It has been standardized for Python as documented in PEPs
-     <a href=\"https://www.python.org/dev/peps/pep-0458/\">458</a> and
-     <a href=\"https://www.python.org/dev/peps/pep-0480/\">480</a>.
-    TUF, and Docker's popular implementation of TUF, are now
-<a href=\"http://www.linuxfoundation.org/\">Linux Foundation</a>
-    projects as part of the Cloud Native Computing Foundation.  
-<a href=\"https://store.cncf.io/collections/tuf\">Buy our merch!</a>"
+<a href=\"https://blog.cloudflare.com/pal-a-container-identity-bootstrapping-tool/\">Cloudflare</a>, and
+<a href=\"https://blog.docker.com/2015/08/content-trust-docker-1-8/\">Docker</a>.
+It has been standardized for Python as documented in PEPs <a href=\"https://www.python.org/dev/peps/pep-0458/\">458</a> and
+<a href=\"https://www.python.org/dev/peps/pep-0480/\">480</a>. And, through CNCF,
+TUF even has <a href=\"https://store.cncf.io/collections/tuf\">merch</a>" for sale.
```

Comment:
Somewhere, possibly here, we need to add more about adopters.  We have so much strength in that regard that we should list at least the major companies.

Reply:
I added several more adoptions from the TUF web site. If these are not the right additions please steer towards where I can find the right ones.